examples/MultipleServoMotors/MultipleServoMotors.ino

```diff
@@ -0,0 +1,58 @@
+/*
+Controling multiple servo motors using attach() and detach()
+by Bhargav Patel <https://github.com/Engineer1999/>
```

Comment:
It would be nice to have a description of the required circuit here (servos on pin 10 and 12).